packages/read-model/README.md

```diff
@@ -0,0 +1,69 @@
+# Shopsys Read Model
+
+[![Build Status](https://travis-ci.org/shopsys/read-model.svg?branch=master)](https://travis-ci.org/shopsys/read-model)
+[![Downloads](https://img.shields.io/packagist/dt/shopsys/read-model.svg)](https://packagist.org/packages/shopsys/read-model)
+
+This bundle for [Shopsys Framework](https://www.shopsys-framework.com) separates templates from model using [read model concept](https://github.com/shopsys/shopsys/blob/master/docs/model/introduction-to-read-model.md).
```

Comment:
```suggestion
This bundle for [Shopsys Framework](https://www.shopsys-framework.com) separates templates from model using [Read Model](/docs/model/introduction-to-read-model.md) concept.
```

Reply:
@vitek-rostislav and the link https://www.shopsys-framework.com still ends up with error. Please change it to http://www.shopsys-framework.com OR https://www.shopsys.com

Reply:
hi, I am not able to do that as the link in the split repository would lead to 404